product_docs/docs/pgd/5/limitations.mdx

```diff
@@ -108,28 +108,26 @@ Be sure to disable transaction streaming when planning to use
 CAMO. You can configure this option globally or in the PGD node group. See
 [Transaction streaming configuration](../transaction-streaming#configuration).
 
--   Not all DDL can run when CAMO is used. If unsupported DDL is used a warning is logged
-and the transactions commit scope is set to local only. The only supported DDL operations are:
-    - non-concurrent CREATE INDEX
-    - non-concurrent DROP INDEX
-    - non-concurrent REINDEX of an individual table or index
-    - CLUSTER (of a single relation or index only)
-    - ANALYZE
-    - TRUNCATE
+-   Not all DDL can run when CAMO is used. If unsupported DDL is used a warning is logged and the transactions commit scope is set to local only. The only supported DDL operations are:
```

Comment:
```suggestion
-   Not all DDL can run when you use CAMO. If you use unsupported DDL, a warning is logged and the transactions commit scope is set to local only. The only supported DDL operations are:
```

Reply:
```suggestion
-   Not all DDL can run when CAMO is used. If unsupported DDL is used a warning is logged and the transactions commit scope is set to local. The only supported DDL operations are:
```
Only is either misplaced (it should go before what it is modifying) or possibly unnecessary.